internal/servers/controller/handlers/accounts/account_service.go

```diff
@@ -322,16 +433,31 @@ func (s Service) deleteFromRepo(ctx context.Context, scopeId, id string) (bool,
 }
 
 func (s Service) listFromRepo(ctx context.Context, authMethodId string) ([]*pb.Account, error) {
-	repo, err := s.repoFn()
+	pwRepo, err := s.pwRepoFn()
 	if err != nil {
 		return nil, err
 	}
-	ul, err := repo.ListAccounts(ctx, authMethodId)
+	pwl, err := pwRepo.ListAccounts(ctx, authMethodId)
```

Comment:
why make the db trip if it's not a password auth method?  I'm curious why you don't have a switch here on subtype?